lib/service/BlockchainModels/AptSellInfo.dart

```diff
@@ -0,0 +1,7 @@
+class AptSellInfo {
```

Comment:
This is nearly the same thing as AptBuyInfo, lets consolidate the models to AptTransactionInfo or something along those lines